proposals/csharp-11.0/low-level-struct-improvements.md

```diff
@@ -869,6 +869,147 @@ ref struct S<T>
 }
 ```
 
+### Annotations
+Lifetime models are most naturally expressed via a more explicit annotation model. While the syntax of C# implicitly adds lifetimes to values, there is an underlying model that describes the fundamental rules here. It's often easier to discuss the implication of changes to the design in terms of these rules so they are included here for discussion sake.
+
+Note that this is not meant to be a 100% complete documentation. Documenting every single behavior isn't a goal here. Instead it's meant to establish a general understanding and common verbiage by which the model, and potential changes to it, can be discussed.
+
+The below examples make use of named lifetimes. The syntax `'a` refers to a lifetime named `a`. It is a lifetime that has no meaning byitself but can be given a relationship to other lifetimes via the `where lifetime` syntax. There are a few predefined lifetimes for convenience and brevity below:
```

Comment:
Consider using $a because it makes the syntax highlighting freak out less when we include it in c# markdown blocks.

Reply:
That's a good suggestion. I started using txt encoding vs. csharp cause the highlighting was so bad.